packages/create-cloudflare/src/types.ts

```diff
@@ -40,6 +40,7 @@ export type PagesGeneratorContext = {
 	};
 	type?: "pages" | "workers";
 	originalCWD: string;
+	c3InitializedGitRepo?: boolean;
```

Comment:
code in C3 usually doesn't use the term `c3` as that does feel rather pointless... here I did so to make super clear that c3 initialized the repo (and that that wasn't done by a framework's cli run by c3 for example), I'm totally happy to use any other name here, if you have suggestions please let me know 🙂 

(just `initializedGitRepo`? `gitRepoInitializedByUs`? etc...)